addons/launcher-addon/src/test/resources/fetch-booster-bundles.sh

```diff
@@ -6,15 +6,11 @@ mkdir -p ${TEST_DIR} > /dev/null 2>&1
 BOOSTER_TMP_DIR="${TEST_DIR}/booster-catalog"
 git clone git@github.com:fabric8-launcher/launcher-booster-catalog.git ${BOOSTER_TMP_DIR}
 cd ${BOOSTER_TMP_DIR}
-for branch in `git branch -a | grep remotes | grep -v HEAD | grep -v master `; do
-   git branch --track ${branch#remotes/origin/} $branch
```

Comment:
Won't this command change to the correct git branch?

Reply:
There is no "correct" git branch anymore, we only have "master" nowadays for the catalog, don't we?

Reply:
Ah, I thought those were for the boosters. In this case, nvm